tools/Deployment/deploy.js

```diff
@@ -434,18 +435,15 @@ switch (branchValue.toLowerCase()) {
       });
     break;
   case "nightly-build":
-    util.runSteps([
-      // step 1: clear the possible release exists
-      clearReleaseStep,
-      // step2: run build.ps1
-      docfxBuildStep,
-      // step3: run e2e test
-      e2eTestStep,
-      // step4: run docfx.exe to generate documentation
-      genereateDocsStep,
-      // step5: upload release to myget.org
-      uploadDevMygetStep
-    ]);
+    promise = util.execPromiseFn("node", [path.join(__dirname, "node_modules/gulp/bin/gulp"), "stable"], __dirname)();
```

Comment:
no need, `deploy.js` would be removed once `gulpfile.js` is finished. We'll run `gulp dev` in teamcity directly instead of this script.

---
In reply to: [101437341](https://github.com/dotnet/docfx/pull/1314#discussion_r101437341) [](ancestors = 101437341)